.github/dependabot.yml

```diff
@@ -9,7 +9,13 @@ updates:
     directory: /
     schedule:
       interval: "weekly"
+    commit-message:
+      # Prefix commit message with "chore: "
+      prefix: "chore"
```

Comment:
This is fine, but where do you want it to show in the Release Notes?

According to our release-drafter config:
- `chore` labels go in the "Maintenance" section.
- `dependencies` labels go in the "Dependency updates" section.

With `git-cliff`, I'm maintaining that distinction:
- `chore:` type commits go in the "Maintenance" section.
- `build:` type commits go in the "Dependency updates" section.

Reply:
The git-cliff config prioritizes PR labels over conventional-commit titles. So, you can write `chore:` and label it with `dependencies`. Then git cliff will only respect the PR label and put it the "Dependency updates" section.

Reply:
If you want, we can add `deps` to the supported list of commit types in committed.toml. I'm not in the habit of using that commit type. The actual [recommended commit types](https://www.conventionalcommits.org/en/v1.0.0/#summary) are pretty limited:

> types other than `fix:` and `feat:` are allowed, for example [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional) (based on the [Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines)) recommends `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others.